Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -35,6 +36,21 @@
 STEP_INIT = "init"
 
 
+def extract_response(response: Any) -> List[Dict[str, Any]]:
+    if response.get("getTableResponse") is None:
+        demisto.error(f"got bad response, {response}")
+        raise Exception(f"got bad response, {response}")
+    else:
+        demisto.info(
+            f"Count of assets exposed publicly: {response.get('getTableResponse').get('totalNumberOfResults')}"
```

Comment:
All the uses of the function are of "assets exposed publicly?"
```suggestion
            f"Count of assets: {response.get('getTableResponse').get('totalNumberOfResults')}"
```